root/etc/e-smith/events/actions/nethserver-collabora-conf

```diff
@@ -44,11 +44,11 @@ if [[ -n `config getprop loolwsd VirtualHost` ]]; then
   #Configure Nextcloud
   if [[ -x "/usr/share/nextcloud/occ" ]]; then
 
-    sudo -u apache scl enable rh-php72 "/usr/share/nextcloud/occ app:install richdocuments"
+    sudo -u apache scl enable rh-php73 "/usr/share/nextcloud/occ app:install richdocuments"
 
-    sudo -u apache scl enable rh-php72 "/usr/share/nextcloud/occ config:app:set richdocuments wopi_url --value=https:\/\/`config getprop loolwsd VirtualHost`"
+    sudo -u apache scl enable rh-php73 "/usr/share/nextcloud/occ config:app:set richdocuments wopi_url --value=https:\/\/`config getprop loolwsd VirtualHost`"
```

Comment:
I remove also the slash escape, but 'm not sure this work, please just try it on machine where collabora is instelled:
```suggestion
    /usr/local/sbin/occ config:app:set richdocuments wopi_url --value=https://`config getprop loolwsd VirtualHost`
```

Reply:
will test this evening.

Reply:
Install was oke